### PR TITLE
Guard against other known limits

### PR DESCRIPTION
**Pull request by KnorpelSenf**
Triggered by https://t.me/tdlibchat/146123

### CODE REVIEW

File: site/docs/advanced/flood.md [hunk 1]
```diff
@@ -64,6 +64,13 @@ From the [Bot FAQ](https://core.telegram.org/bots/faq#my-bot-is-hitting-limits-h
    Completely unrelated to bulk notifications or how many messages are sent in the group.
    And yet again, the auto-retry plugin will take care of this for you.
 
+There are a few other known limits there were revealed outside of the official Bot API documentation.
+For example, [it is known](https://t.me/tdlibchat/146123) that bots can only do up to 20 message edits in a minute per group chat.
+However, this is the exception, and we also have to assume that these limits may be changed in the future.
+Thus, this information does not affect how to program your bot.
+
+For instance, throttling your bot based on these number is still a bad idea:
```
Comment (roj1512) [inline site/docs/advanced/flood.md:72]: What is that colon pointing to?

Comment (KnorpelSenf) [reply site/docs/advanced/flood.md:72]: The next section

**Review by deptyped — APPROVED** [review]

File: site/docs/id/advanced/flood.md [hunk 2]
```diff
@@ -63,6 +63,13 @@ Berdasarkan [FAQ Bot](https://core.telegram.org/bots/faq#my-bot-is-hitting-limit
    Yang ini sama sekali tidak ada hubungannya dengan notifikasi masal ataupun banyaknya pesan yang dikirim di dalam grup.
    Sekali lagi, plugin auto-retry akan menanganinya dengan baik.
 
+Ada beberapa batasan lain yang diketahui dan terungkap di luar dokumentasi API Bot resmi.
+Contohnya, [yang diketahui](https://t.me/tdlibchat/146123) bahwa bot hanya dapat mengedit 20 pesan dalam satu menit per obrolan grup.
+Namun, ini hanyalah pengecualian, kita dapat asumsikan bahwa batasan ini bisa saja berubah di masa mendatang.
+Lagi pula, informasi ini tidak memberikan dampak apapun pada cara Anda memprogram bot.
+
+Misalnya, memperlambat bot kamu berdasarkan angka ini tetaplah ide yang buruk:
+
```
Comment (quadratz) [inline site/docs/id/advanced/flood.md:72]:
```suggestion
Ada beberapa batasan lain yang tidak tertulis di dokumentasi resmi API Bot.
Contohnya, [sudah diketahui](https://t.me/tdlibchat/146123) bahwa bot hanya dapat mengedit maksimal 20 pesan dalam satu menit per obrolan grup.
Namun, batasan ini tidaklah selamanya benar, karena ia bisa saja berubah di masa mendatang.
Lagi pula, informasi ini tidak memberikan dampak apapun ke cara kamu memprogram bot.

Misalnya, memperlambat bot kamu berdasarkan angka di atas tetaplah ide yang buruk:

```

**Review by quadratz — CHANGES_REQUESTED** [review]

**Review by dzikrl — APPROVED** [review]

**Review by quadratz — APPROVED** [review]

File: site/docs/advanced/flood.md (hunk 1 above)
Comment (roziscoding) [inline site/docs/advanced/flood.md:72]:
```suggestion
For instance, throttling your bot based on this number is still a bad idea:
```

**Review by roziscoding — APPROVED** [review]

File: site/docs/advanced/flood.md (hunk 1 above)
Comment (KnorpelSenf) [reply site/docs/advanced/flood.md:72]: Nice catch, I actually meant
```suggestion
For instance, throttling your bot based on these numbers is still a bad idea:
```

**Review by roziscoding — APPROVED** [review]

**Pull request merged**